nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-process-group-configuration.js

```diff
@@ -348,7 +354,10 @@
         var parameterContexts = $.ajax({
                 type: 'GET',
                 url: config.urls.parameterContexts,
-                dataType: 'json'
+                dataType: 'json',
+                data: {
+                    uiOnly: true
```

Comment:
The parameter contexts endpoint does not support the `uiOnly` parameter. Was this added inadvertently?

Reply:
Oops, good catch. That was definitely inadvertently